### PR TITLE
🐛 Source Hubspot: Use correct schema for 'owners' stream

### LINKED ISSUE #6147: Hubspot connector: Use correct schema for owners table
**Enviroment**

**Is this your first time deploying Airbyte:** No
**OS Version / Instance:** AWS
**Deployment:** EC2
**Airbyte Version:** 0.29.17-alpha
**Source name/version:** Hubspot 0.1.13
**Destination name/version:** Postgres 0.3.11
**Severity**: High
**Step**: Sync
**Description/ Current Behavior:** fields _type, ownerid, isactive, portalid, signature, remotelist, activeuserid, hascontactaccess, useridincludinginactive_ have null for all records. 

**Expected Behavior:**
fields _type, ownerid, isactive, portalid, signature, remotelist, activeuserid, hascontactaccess, useridincludinginactive_ have values for all records.

**Logs:**
No error in logs, just missing fields.
example of airbyte_data field in raw table 
{"id", "email", "teams": [{"id", "name"}], "userId", "archived", "lastName", "createdAt", "firstName", "updatedAt"}

**Steps to Reproduce**
Setup a sync with the Hubspot connector on an account that has Owners data.
Run a sync
multiple fields mentioned in the description will not be retrieved.

### PR DESCRIPTION
**Pull request by yevhenii-ldv**
## What
*resolves #6147.*

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

### CODE REVIEW

File: docs/integrations/sources/hubspot.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ This connector supports only authentication with API Key. To obtain API key for
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.15   | 2021-09-?? | [????](https://github.com/airbytehq/airbyte/pull/????) | Use correct schema for `owners` stream |
```
Comment (avida) [inline docs/integrations/sources/hubspot.md:76]: Please update ????

**Review by avida — APPROVED** [review]

**Review by Zirochkaa — APPROVED** [review]

File: docs/integrations/sources/hubspot.md (hunk 1 above)
Comment (yevhenii-ldv) [reply docs/integrations/sources/hubspot.md:76]: I'll update this at the version bumping stage before the connector is released, along with the release date.

File: airbyte-integrations/builds.md [hunk 2]
```diff
@@ -36,7 +36,7 @@
...
 | Google Workspace Admin   | [![source-google-workspace-admin-reports](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-google-workspace-admin-reports%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-google-workspace-admin-reports) |
 | Greenhouse   | [![source-greenhouse](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-greenhouse%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-greenhouse) |
-| Hubspot   | [![source-hubspot-singer](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-hubspot%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-hubspot) |
+| Hubspot   | [![source-hubspot](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-hubspot%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-hubspot) |
```
Comment (sherifnada) [inline airbyte-integrations/builds.md:39]: thanks!

**Review by sherifnada — APPROVED** [review]

**Pull request merged**